forge/forge/python_codegen.py

```diff
@@ -53,6 +55,51 @@ def forge_df_str_from_str(df: str, name: str):
     else:
         logger.warning(f"Invalid data format: {df} for constant/parameter '{name}', defaulting to float32")
         return "forge.DataFormat.Float32"
+    
+def forge_df_from_str(df: str):
```

Comment:
Can we merge this one and the function above? Seems like they can be single-function pretty easily